### ISSUE TITLE
[BUG] Too many new chapters even if they are not of today

### ISSUE THREAD

**Issue by DeeJack** [inorichi/tachiyomi#1759]
**App version:** 0.8.0

**Issue/Request:** Every time I add a manga to the library, if I reload the "library updates" tab, it add too many chapters even though they aren't of today. Sorry for the bad english, I'll leave a photo to explain better.

**Steps to reproduce (if applicable)**

 1. Add a manga to the library
 2. Reload the "library update" tab
 3. See what appens

**Other details:**
![photo_2018-12-03_19-15-03](https://user-images.githubusercontent.com/20882533/49392783-c1b22000-f72f-11e8-8c35-908ed1d93182.jpg)
![screenshot_2018-12-03-19-03-58](https://user-images.githubusercontent.com/20882533/49392784-c24ab680-f72f-11e8-8ef6-33964d462121.jpg)

**Comment by bbtora** [comment]
Would also like to mention a similar issue #1374.

**Comment by hXtreme** [comment]
Are you by any chance adding the manga to favourites by long pressing or are you using the button on the manga details page?

**Comment by DeeJack** [comment]
Usually I add a manga after reading the plot, so by using the button on the details page.

**Comment by hXtreme** [comment]
I see. Also, does this happen for any manga source or only specific ones?

**Comment by phanirithvij** [comment]
Happened to me too and that's a fine screenshot.

**Comment by hXtreme** [comment]
I think I've figured out what causes this behaviour and *if* I'm correct then I don't see any direct fixes.

If the following actions occur in the order shown below then, the library should get an extra updates.

START   
You click on a Manga.  
App opens the summary page.  
App starts downloading info.  
App loads up summary, other details and starts parsing for chapter info.  
You click the bookmark button.  
App adds the manga to library.
App finishes parsing the chapters and loads up the chapters.   
END   

In this case, chapters that are loaded up in the very last step should show up in the updates. This should for the most part not have been a problem if the number of chapters is small or if the download speed is fast enough.

Furthermore, from the screenshots, it looks like the effect became more prominent because more than one (Hunter Age & Martial Legacy) (both around 100 chapters) mangas were added fairly close to one another.
